src/signals/incident/containers/IncidentReplyContainer/__tests__/IncidentReplyContainer.test.tsx

```diff
@@ -47,7 +47,7 @@ describe('IncidentReplyContainer', () => {
       await waitFor(() => {
         screen.getByRole('heading', { name: 'Aanvullende informatie' })
         screen.getByRole('heading', { name: 'Uw melding' })
-        screen.getByText('Nummer: SIA-1234')
```

Comment:
I'm not sure if this particular format is used anywhere else in the whole process of incident management. Can we maybe move the `SIA-` prefix to the configuration?

Reply:
You mean, the `SIA-` prefix is used in other parts of the app, for Amsterdam, but configurable? And so it should be

Reply:
The prefix is indeed used in the administration of Amsterdam, so it needs to be configurable.